packages/terra-form-select/src/multiple/Frame.jsx

```diff
@@ -223,7 +228,7 @@ class Frame extends React.Component {
             </li>
           ) : null}
         <li className={cx('search-wrapper')}>
-          <input {...inputAttrs} value={searchValue} />
+          {(this.props.isSearchDisabled) ? <div className={cx('placeholder')}>{placeholder || '\xa0'}</div> : <input {...inputAttrs} value={searchValue} /> }
```

Comment:
Keyboard navigation seems broken.

![multiple-select](https://user-images.githubusercontent.com/10090859/68657095-0c036a00-0559-11ea-8fac-abe2dda68d6b.gif)

Reply:
it was due to placeholder div fixed it : 148aaf18db9acf04ced1b29d35d98c3ada852ee4